### ISSUE TITLE
No body with method options

### ISSUE THREAD

**Issue by buschhardt** [rest-client/rest-client#542]
Hallo, I send a request to my REST server with method options, but the body is empty. I check the response with tcpdump, but there is the correct answer.

```
Request: OPTIONS /ansible
Response (Body): ["myfunction"]
```

but the rest-client:

```
RestClient.options(host+'/ansible')
<RestClient::Response 200 "">
```

Thank You

PS: I cant attach the Wireshark dumpfile here. But here is a link http://www.file-upload.net/download-12051243/output2.dump.html

**Comment by ab** [comment]
What is the `.length` on the response object? And what is `.headers[:content_length]`?

**Comment by buschhardt** [comment]
Hallo
```
a=RestClient.options(host+'/ansible')
a.length
=> 0
a.headers[:content_length]
=> "5"
```
Thank You

**Comment by ab** [comment]
Very interesting. I think this may be a Ruby Net::HTTP bug. I'm able to reproduce the problem on Ruby 2.0 and Ruby 2.1, but it seems to have been fixed for Ruby 2.2 and Ruby 2.3, where I can't reproduce it. Have you tried on newer versions of Ruby?

```
ruby 2.0.0p648 (2015-12-16 revision 53162) [x86_64-linux]
0	""
```

```
ruby 2.1.10p492 (2016-04-01 revision 54464) [x86_64-linux]
0	""
```

```
ruby 2.2.5p319 (2016-04-26 revision 54774) [x86_64-linux]
14	"[\"myfunction\"]"
```

```
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-linux]
14	"[\"myfunction\"]"
```

**Comment by ab** [comment]
Closing under the assumption that this is a ruby bug, but feel free to reopen with more info.
